spring-cloud-gcp-autoconfigure/src/main/java/org/springframework/cloud/gcp/autoconfigure/pubsub/GcpPubSubProperties.java

```diff
@@ -21,14 +21,21 @@
 import org.springframework.cloud.gcp.core.Credentials;
 import org.springframework.cloud.gcp.core.GcpScope;
 
+/**
+ * @author João André Martins
+ */
 @ConfigurationProperties("spring.cloud.gcp.pubsub")
 public class GcpPubSubProperties {
+	/** Number of threads used by every {@link com.google.cloud.pubsub.v1.Subscriber}. Default is 4 */
```

Comment:
Please don't add "Default is" in the description. The metadata already has that information and the IDE displays it.

I usually use the IDE with the keys that I document to make sure that it has the right developer experience.